test-suite-groovy/src/test/groovy/io/micronaut/docs/server/uris/UriTemplateTest.groovy

```diff
@@ -15,4 +15,14 @@ class UriTemplateTest extends Specification {
         template.expand(["name": "John"]) == "/hello/John" // <2>
         // end::match[]
     }
+
+    void "test uri with slash before params"() {
+        // tag::match[]
```

Comment:
Could you remove these `tag:`` and `end::`` comments from the new test as they may break documentation